### DISCUSSION TITLE
Search not working, need some help

### DISCUSSION THREAD

**Discussion by bertmelis** [immich-app/immich#5802] [Q&A]
It's probably a configuration error, but I honestly don't where to start looking for the culprit:

when I type something in the search bar, I get a 500 error. The server log says this:
```
[Nest] 2  - 12/17/2023, 8:06:32 PM   ERROR [ErrorInterceptor] Failed to search
[Nest] 2  - 12/17/2023, 8:06:32 PM   ERROR [ErrorInterceptor] TypeError: fetch failed
```

**Comment by alextran1502** [comment]
What version are you on? metadata search is broken on some instance. This PR fixed it #5800

**Reply by bertmelis** [reply]
I just pulled latest. I was manually upgrading because of the change in stack.

**Reply by alextran1502** [reply]
text/clip search should still work. What was the search mechanism did you use?

**Reply by bertmelis** [reply]
I'm on 1.19.3.

I just typed something in the search bar ("rollercoaster") and pressed enter. Next thing was the 500 error.

**Reply by alextran1502** [reply]
Hmm, are you still using Podman? Can you post your configuration?

**Reply by bertmelis** [reply]
any particular container? I manually started the stack, raw commands. so no compose file.

**Reply by bertmelis** [reply]
Mind that searching using `m:rollercoaster` works as expected (no results though).

**Reply by alextran1502** [reply]
Do you have machine learning enabled? Is your machine learning container running?

**Reply by bertmelis** [reply]
machine learning is running. I probably need to set the env variable to the ml host? In podman this isn't 0.0.0.0. They talk to each other using the container name as hostname. I'll check redis too.

**Reply by alextran1502** [reply]
The ML host can be set in the machine learning setting.

![image](https://github.com/immich-app/immich/assets/27055614/b16fa4ba-42ea-4c6c-960d-2767de25a8e8)

**Comment by bertmelis** [comment]
I don't know if it's related, but my postgres container gives this log:

```
2023-12-17 20:00:52.526 UTC [471] ERROR:  relation "smart_search" already exists
2023-12-17 20:00:52.526 UTC [471] STATEMENT:
                CREATE TABLE smart_search (
                "assetId"  uuid PRIMARY KEY NOT NULL REFERENCES assets(id) ON DELETE CASCADE,
                embedding  vector(512) NOT NULL )
```

(I actually don't know if this belongs to Immich. It's a shared instance)

**Reply by alextran1502** [reply]
We fixed this in `1.91.3`, released a couple of hours ago

**Reply by bertmelis** [reply]
Hmmm. I'm going to restart and see if it is still there. Because I thought I just pulled 1.91.3.

**Reply by bertmelis** [reply]
Ok, this is not appearing anymore.

**Comment by NicholasFlamy** [comment]
Closing because this is old and it seems this was resolved.
